pkg/kubelet/server/stats/summary.go

```diff
@@ -83,6 +83,10 @@ func (sp *summaryProviderImpl) Get() (*statsapi.Summary, error) {
 		statsapi.SystemContainerMisc:    nodeConfig.SystemCgroupsName,
 	}
 	for sys, name := range systemContainers {
+		// skip if cgroup name is undefined (not all system containers are required)
```

Comment:
Which one is not required?

Reply:
I think whichever is not defined in the config.